### DISCUSSION TITLE
Codeql Database analyze

### DISCUSSION THREAD

**Discussion by sam-cha-tfs** [github/codeql#13060] [General]
When running the following command with no query pack specified what is the default query pack that gets used?

`codeql database analyze <database> --format=<format> --output=<output>`

**Comment by aibaars** [comment]
The default packs have names like `codeql/LANG-queries` and can be found on https://github.com/orgs/codeql/packages/. For example [`codeql/java-queries`](https://github.com/orgs/codeql/packages/container/package/java-queries).
